src/utils.js

```diff
@@ -0,0 +1,32 @@
+import dayjs from 'dayjs';
+
+export const getRandomNumber = (min, max) => {
+  if (min < max && min >= 0) {
+    const randomNumber = Math.random() * ((max + 1) - min) + min;
+
+    return Math.floor(randomNumber);
+  } throw new Error('Диапазон задан неверно: минимальное значение диапазона должно быть меньше максимального. Граничные значения диапазона не могут быть меньше 0.');
```

Comment:
Необходимо разместить на след строке 
```suggestion
  } 
  throw new Error('Диапазон задан неверно: минимальное значение диапазона должно быть меньше максимального. Граничные значения диапазона не могут быть меньше 0.');
```

Reply:
принято